tests/blockchain/eth/contracts/integration/test_intercontract_integration.py

```diff
@@ -37,25 +37,9 @@
 NULL_ADDR = '0x' + '0' * 40
 
 VALUE_FIELD = 0
-DECIMALS_FIELD = 1
-CONFIRMED_PERIOD_1_FIELD = 2
-CONFIRMED_PERIOD_2_FIELD = 3
-LAST_ACTIVE_PERIOD_FIELD = 4
-
-CLIENT_FIELD = 0
-RATE_FIELD = 1
-FIRST_REWARD_FIELD = 2
-START_PERIOD_FIELD = 3
-LAST_PERIOD_FIELD = 4
-DISABLED_FIELD = 5
-
-REWARD_FIELD = 0
-REWARD_RATE_FIELD = 1
-LAST_MINED_PERIOD_FIELD = 2
+RE_STAKE_FIELD = 3
```

Comment:
Is this changing the structure of `stakeInfo`?

Reply:
No, just removed old unused parts